src/csharp/Grpc.Examples/MathGrpc.cs

```diff
@@ -27,10 +27,77 @@ public static partial class Math
   {
     static readonly string __ServiceName = "math.Math";
 
+    #if !GOOGLE_PROTOBUF_DISABLE_BUFFER_SERIALIZATION
+    static readonly grpc::Marshaller<global::Math.DivArgs> __Marshaller_math_DivArgs = 
+      grpc::Marshallers.Create(
+        (arg, context) =>
+        {
+          var writer = new global::Google.Protobuf.CodedOutputWriter(context.GetBufferWriter());
+          arg.WriteTo(ref writer);
+          writer.Flush();
```

Comment:
Seems like we need to flush, but is this the right place?

Reply:
Yes. It commits any outstanding content in the coded output writer to the underlying buffer writer.